CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 [Unreleased]
 * Added _UnityEngine.Object.FindObjectOfType_ usage detection
 * Fixed names of build-generated assets in Build Report 
+* Added FogModeAnalyser module
```

Comment:
Small typo:  "FogModeAnalyzer" is written with a "z" in code (that is just American English, instead of British English with an "s")